src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -92,6 +92,12 @@ class DocumentSerializer(serializers.Serializer):
             "typically *before* the registration date."
         )
     )
+    gepubliceerd_op = serializers.DateTimeField(
+        help_text=_(
+            "System timestamp reflecting when the document was published in the "
+            "GPP-Publicatiebank."
+        )
+    )
```

Comment:
To not need another breaking change release, this field should not be required.


```suggestion
    gepubliceerd_op = serializers.DateTimeField(
        required=False,
        allow_null=True,
        help_text=_(
            "System timestamp reflecting when the document was published in the "
            "GPP-Publicatiebank."
        )
    )
```